python/setup.py

```diff
@@ -279,6 +279,7 @@ def get_packages(self):
         "frozenlist",
         "requests",
         "virtualenv",  # For pip runtime env.
+        "async-timeout",  # For runtime_env_agent set timeout.
```

Comment:
@architkulkarni @edoakes @richardliaw Hi, ray requires aiohttp >= 3.7, but the async-timeout version ranges required by aiohttp 3.7 (async_timeout>=3.0,<4.0) and aiohttp 3.8 ( async_timeout >= 4.0.0a3, < 5.0) do not intersect, so if I copy async-timeout's source code, then using the copied async-timeout and aiohttp at the same time may have the risk of potential conflict

Reply:
That's unfortunate... In that case, looks like the best solution is to just include `async_timeout` in `setup.py` and `requirements.txt`.  

@richardliaw Thoughts?  It has no dependencies, and the wheel is 5.8 kB.

Reply:
I think the overhead is low and acceptable. Let's just include it?

Reply:
Why do we need to use `async_timeout` instead of just using `asyncio.wait_for`?
https://docs.python.org/3/library/asyncio-task.html#asyncio.wait_for

We should be very careful about adding any new dependencies unnecessarily.

Reply:
> Why do we need to use `async_timeout` instead of just using `asyncio.wait_for`? https://docs.python.org/3/library/asyncio-task.html#asyncio.wait_for
> 
> We should be very careful about adding any new dependencies unnecessarily.

The reason I didn't use `asyncio.wait_for` before was that I thought async_timeout would work with synchronous operations, it turns out I thought too much, in this example, async_timeout doesn't work:
```python
async def test():
    from time import sleep
    sleep(3600)
with async_timeout.timeout(10):
    await test()
```
If timeout can work in synchronous operation, it can improve security, I checked the code, and currently, all our operations that may block the loop are in a new thread, so you're right, `asyncio.wait_for` should be used here.

Reply:
here is why aiohttp use `async_timeout`: https://stackoverflow.com/questions/47000770/async-timeout-timeout-vs-asyncio-wait-for?answertab=scoredesc#tab-top

It seems that aiohttp is more concerned about performance issues, while runtime_env_agent is not performance sensitive